sample-apps/s3-java/README.md

```diff
@@ -63,10 +63,14 @@ You can also build the application with Maven. To use maven, add `mvn` to the co
     ...
 
 # Test
-To upload an image file to the application bucket and trigger the function, run `4-upload.sh`.
+This Lambda function takes an image that's currently stored in S3, and scales it down into
+a thumbnail-sized image. To upload an image file to the application bucket, run `4-upload.sh`.
 
     s3-java$ ./4-upload.sh
 
+In your `s3-java-bucket-<random_uuid>` bucket that was created in step 3, you should now see a
```

Comment:
The bucket name starts with `lambda-artifacts-`.

Reply:
I think in the 4-upload.sh script, it's describing the stack to determine the correct bucket to upload the image to, which is the s3-java-bucket-<random_uuid> bucket.

Reply:
I see, you're correct. I saw the images in the artifact bucket but now I realize that those are from the test class. Thanks for confirming.